src/widgets/_checkbuttons.scss

```diff
@@ -62,6 +62,15 @@ checkbutton {
             margin-right: rem(6px);
         }
     }
+
+    &.image-button:checked {
+        background: #{'alpha(@accent_color, 0.1)'};
+        border-radius: rem(6px);
```

Comment:
```suggestion
        border: 1px solid #{'alpha(@accent_color, 0.25)'};
        border-radius: rem(3px);
```

Reply:
I'm not too picky about this border-radius, but assumed the larger radius was in part due to the lack of a stroke. I think 3px matches what we use elsewhere?